wolfcrypt/src/sp_x86_64_asm.S

```diff
@@ -20,7 +20,7 @@
  */
 
 #ifdef WOLFSSL_USER_SETTINGS
-#include "wolfssl/wolfcrypt/settings.h"
+#include "wolfssl/wolfcrpyt/settings.h"
```

Comment:
Not sure what happened here... `wolfcrypt` -> `wolfcrpyt`? Guessing this line needs reverted.

Reply:
Fixed here: https://github.com/wolfSSL/scripts/pull/245

Reply:
Changed script but code got checked in by someone else.
Fixed now.